src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -2254,9 +2254,14 @@ def _ensure_osa_aad(cli_ctx,
         # Delegate Sign In and Read User Profile permissions on Windows Azure Active Directory API
         resource_access = ResourceAccess(id="311a71cc-e848-46a1-bdf8-97ff7156d8e6",
                                          additional_properties=None, type="Scope")
-        required_osa_aad_access = RequiredResourceAccess(resource_access=[resource_access],
+        # Read directory permission on Windws Azure Active Directory API
+        directory_access = ResourceAccess(id="5778995a-e1bf-45b8-affa-663a9f3f4d04",
```

Comment:
Just curious what is this ?

Reply:
@julienstroheker: "5778995a-e1bf-45b8-affa-663a9f3f4d04" -> Read directory data